expo-plugin/src/android/install_reclaim_android_manifest.ts

```
@@ -0,0 +1,44 @@
+import { type ConfigPlugin, withAndroidManifest } from '@expo/config-plugins';
+
+export const installReclaimAndroidManifest: ConfigPlugin = (config) => {
+    return withAndroidManifest(config, (config) => {
```

ESLint

[error] Delete `··`

(prettier/prettier)